File: 05/Rigon/03.c
/* Escreva um programa que leia via e imprima na tela um vetor X de 10
elementos. O programa deve imprimir um vetor Y obtido pela inversão da
ordem de 10 elementos de X.  */

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(){
  int i = 0, j = 0;
  float X[10], Y[10];

  for(i=0;i<10;i++)
    scanf("%f", &X[i]);

  for(i=0,j=9;i < 10;i++,j--)
    Y[j] = X[i];

  for(i=0;i<=10;i++)
    printf("%d: %.2f\n", i, Y[i]);

  return 0;
}

File: 05/Rigon/python/03.py
# -*- coding: utf-8 -*-
'''
/* Escreva um programa que leia via e imprima na tela um vetor X de 10
elementos. O programa deve imprimir um vetor Y obtido pela inversão da
ordem de 10 elementos de X.  */

'''
X = [] * 10
print("Insira 10 valores")
for i in range(10):
    n = float(input())
    X.append(n)

print("Vetor original:")
print(X)

j = 9
Y = [] * 10

for i in range(10):
    Y.append(X[j])
    j -= 1
print("Vetor invertido:")
print(Y)
